src/panels/lovelace/cards/energy/common/energy-chart-options.ts

```diff
@@ -40,12 +40,12 @@ export function getSuggestedMax(dayDifference: number, end: Date): number {
     suggestedMax = subHours(suggestedMax, 1);
   }
 
-  suggestedMax.setMinutes(0, 0, 0);
-  if (dayDifference > 35) {
-    suggestedMax.setDate(1);
-  }
   if (dayDifference > 2) {
     suggestedMax.setHours(0);
+    suggestedMax.setMinutes(0, 0, 0);
+    if (dayDifference > 35) {
+      suggestedMax.setDate(1);
+    }
   }
```

Comment:
```suggestion
if (dayDifference > 35) {
    suggestedMax.setDate(1);
}
if (dayDifference > 2) {
    suggestedMax.setHours(0);
    suggestedMax.setMinutes(0, 0, 0);
}
```
Flatten conditions we don't have nested conditions (easier to read)

Reply:
It's not easier for me but alright